src/react-app-env.d.ts

```diff
@@ -1,2 +1,12 @@
 // eslint-disable-next-line
 /// <reference types="react-scripts" />
+interface People {
+  name: string;
+  sex: 'm' | 'f';
+  born: number;
+  died: number;
+  fatherName: string;
+  motherName: string;
+  mother: Person | null;
+  father: Person | null;
```

Comment:
Where do you have this `Person` type?

Reply:
you should have two interface
- the first one will have `motherName` and `fatherName`, just as strings (e.g. `PersonaData`)
- the second one will have `mother` and `father`, as objects of type `PersonData` (e.g. `Person`)